README.md

```diff
@@ -1,2 +1,18 @@
-# bootcamp_3_api_tests
+# Bootcamp API automated tests
+
 Automated API tests to cover endpoints from https://reqres.in/ resource
+
```

Comment:
Changeable variables in the documentation are marked by {}.
Please, replace 'https://reqres.in/ resource' to 'https://reqres.in/{resource}'